winrm_plugin/example-script/script.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+
+def write():
+    try:
+        file = open('c:\\users\\administrator\\desktop\\python-succes.txt',
+                    'a')
+        file.close()
+
+    except:
+        print('Something went wrong! Can\'t tell what?')
```

Comment:
Use this:

http://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python